src/activities/content/ContentFileEntity.js

```diff
@@ -127,4 +131,17 @@ export class ContentHtmlFileEntity extends ContentWorkingCopyEntity {
 	getFileHref() {
 		return ContentHelperFunctions.getHrefFromRel(Rels.Files.file, this._entity);
 	}
+
+	/**
+	 * @returns {string|null} Returns the File type
+	 */
+	getFileType() {

```

Comment:
The file type is determined by the class in the file link:
```
{
      "class": [
        "file",
        "html"
      ],
      "rel": [
        "https://files.api.brightspace.com/rels/file"
      ],
      "type": "application/vnd.siren+json",
      "href": "https://2864b37c-fb9f-4b5b-ae27-897885ef47ce.files.api.proddev.d2l/6606_645000_6613_File%20(HTML).html/usages/6613"
}
```
...all files will have the `file` class, and the specific file type will be determined by a second class. As of now, we only have the `html` class.